geode-protobuf/src/integrationTest/java/org/apache/geode/internal/protocol/protobuf/v1/acceptance/CacheConnectionJUnitTest.java

```diff
@@ -82,6 +82,12 @@
   private final String TEST_VALUE = "testValue";
   private final String TEST_REGION = "testRegion";
 
+  /*
```

Comment:
We can start using `CertStores` utility to dynamically generate keystores and truststores instead of adding them as resources, but that can be part of another refactoring.